### PR TITLE
Update: SVG workand clean up on about pref page

### LINKED ISSUE #6931: Rework about:pref page SVG icons and clean up css/dom
- Did you search for similar issues before submitting this one?
yes

- Describe the issue you encountered:
Rework about:pref page SVG icons and clean up css/dom. Make it more responsive.

- Platform (Win7, 8, 10? macOS? Linux distro?):
all
- Brave Version (revision SHA):
master
- Steps to reproduce:
    n/a
    
- Actual result:
n/a
- Expected result:
n/a
- Will the steps above reproduce in a fresh profile? If not what other info can be added?
n/a
- Is this an issue in the currently released version?
n/a
- Can this issue be consistently reproduced?
n/a
- Extra QA steps:
    n/a.

- Screenshot if needed:
n/a
- Any related issues:
n/a

### PR DESCRIPTION
**Pull request by gyandeeps**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Test Plan:
fixes #6931
This is cleanup work to consume svg icons on the about:pref page.

screen shot 2017-02-15 at 7 31 14 pm

### CODE REVIEW

**Comment by luixxiul** [comment]
Thanks for your hard work. Would you please open an issue or associate this PR to the existing one if any so @bradleyrichter and @bsclifton would review the PR?

**Comment by luixxiul** [comment]
And I am afraid this work woud need refactoring to Aphrodite (to avoid the duplicate work). This is the guideline that might be worth reading if you have not: https://github.com/brave/browser-laptop/blob/master/docs/style.md

CC @bsclifton for an advice

**Comment by bsclifton** [comment]
@gyandeeps I can help you make this Aphrodite friendly 😄  I should be able to review this soon (hang in there!)

**Comment by cezaraugusto** [comment]
@gyandeeps nice work here. Based on @bsclifton request, we have a new wiki page regarding how to refactor styles which may help you:

https://github.com/brave/browser-laptop/wiki/Refactoring-styles-to-Aphrodite

**Comment by cezaraugusto** [comment]
**Note:**
Needs resolution about sync tab (disabled on this PR at this moment). /cc @bradleyrichter.

Split prefs nav components, refactored to Aphrodite, rebased. Ready for code review.

Thanks @gyandeeps for the great initial work. Way better overall styles and flexbox usage. 🎉

File: js/about/preferences.js [hunk 1]
```diff
@@ -9,12 +9,12 @@ const Immutable = require('immutable')
 const UrlUtil = require('../lib/urlutil')
 
 // Components
+const PreferenceNavigation = require('../../app/renderer/components/preferences/PreferenceNavigation')
 const ModalOverlay = require('../components/modalOverlay')
 const {SettingsList, SettingItem, SettingCheckbox, SiteSettingCheckbox} = require('../../app/renderer/components/settings')
 const {SettingTextbox} = require('../../app/renderer/components/textbox')
 const {SettingDropdown} = require('../../app/renderer/components/dropdown')
 const Button = require('../components/button')
-const HelpfulHints = require('../../app/renderer/components/preferences/helpfulHints')
```
Comment (luixxiul) [inline js/about/preferences.js]: @cezaraugusto should this be deleted?

Comment (cezaraugusto) [inline js/about/preferences.js]: yes it [migrated](https://github.com/brave/browser-laptop/pull/6904/files#diff-762c1a1aeffa016c8990d5219c385df6R75) to `preferencesNavigation`

**Comment by gyandeeps** [comment]
@cezaraugusto Latest changes broke the change I made, where i moved the lower hint text block to always be at the bottom annd on resize stick to the bottom.

Current:

image

Expected:

https://bravesoftware.slack.com/files/bradrichter/F3XQDKR0X/pasted_image_at_2017_01_26_01_43_pm.png

**Comment by bradleyrichter** [comment]
That was a nice improvement. Need to keep that part of the PR working.

File: app/renderer/components/preferences/helpfulHints.js [hunk 2]
```diff
@@ -40,11 +40,15 @@ const styles = StyleSheet.create({
   },
 
   helpfulHints: {
-    width: '100%',
-    height: '100%',
     cursor: 'default',
-    padding: '30px 0',
-    marginTop: '20px'
+    padding: '20px 0 0',
+    overflowY: 'auto',
```
Comment (cezaraugusto) [inline app/renderer/components/preferences/helpfulHints.js]: adding sync and plugins tabs made menu taller, so helpfulHints got cut.

For small displays it adds a scrollbar to address that

File: app/renderer/components/preferences/helpfulHints.js [hunk 3]
```diff
@@ -40,11 +40,15 @@ const styles = StyleSheet.create({
...
+
+    '@media (min-height: 620px)': {
```
Comment (cezaraugusto) [inline app/renderer/components/preferences/helpfulHints.js]: if we have enough display height then put hints at screen's bottom.

Ideally we should do it on navigation itself but then we lost the active indicator (arrow)

**Comment by cezaraugusto** [comment]
@gyandeeps @bradleyrichter sorry I was unaware of those changes. Updated.

**Pull request merged**